devel/cpp-argparse-dev/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+github.setup            rue-ryuzaki argparse 1.6.5 v
+github.tarball_from     archive
+revision                0
+name                    cpp-argparse-dev
+categories              devel
+platforms               darwin
+supported_archs         noarch
```

Comment:
```suggestion
```
This project compiles architecture-specific files, and so this shouldn't be set.